events/wc_druidic_events.txt

```diff
@@ -82,10 +114,16 @@ character_event = {
 						is_druid_class_trigger = yes
 						is_loa_priest_class_trigger = yes
 					}
-					is_5_6_magic_class_trigger = yes
```

Comment:
It's here to check level. Class is checked above. Plus, I wouldn't make it for druids and mages only. I can imagine shaman or loa priest doing it.